src/storage/StorageFlags.h

```diff
@@ -19,6 +19,8 @@ DECLARE_int32(waiting_new_leader_retry_times);
 
 DECLARE_int32(waiting_new_leader_interval_in_secs);
 
+DECLARE_int32(rebuild_index_batch_size);
+
 DECLARE_bool(ignore_index_check_pre_insert);
```

Comment:
Remove the flag

Reply:
I put it into another PR [Remove ignore_index_check_pre_insert flag](https://github.com/vesoft-inc/nebula/pull/1792)